### ISSUE TITLE
Fix specificity bullet list and clarify rules in CSS Foundations lesson

### ISSUE THREAD

**Issue by dm-murphy** [TheOdinProject/curriculum#22975]
The first CSS Foundations lesson (https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/css-foundations) includes some confusion in explaining the rules of specificity in CSS. This issue was discussed on the Discord thread `#html-css-rewrite-discussion` with Mr. ARL and 'thatblindgeye': 'Eric'.

Currently the lesson includes a bullet list on selector specificity as sourced from MDN. But the TOP list also includes a 4th bullet item on universal selectors and combinator symbols. 
```
1. ID selectors (most specific selector)
2. Class selectors
3. Type selectors
4. Universal selector and combinators (no specificity)
```
The 4th bullet should have just been a note on how these combinators  (`+, ~, >`, and a single empty space) as well as the universal selector `*` do not add specificity in and of themselves. It has caused learner confusion by ordering it as a 4th bullet item because it incorrectly implies that combinations would have no specificity at all. This current bullet list is problematic since the first code example after this list involves a single class selector against a combination of two class selectors. So the note on "no specificity" has caused multiple learners confusion in the last week.

I propose a fix to remove this 4th bullet item, clarify its content as a separate note and offer up additional examples on CSS selectors to show how this all works. The fix may or may not include a link to a CSS Specificity Calculator as well. 

Any additional thoughts on this are welcome and thanks for considering this issue!

**Comment by thatblindgeye** [comment]
@dm-murphy Just wanted to check in since it's been a lil bit. Did you still want to take this issue on?

**Comment by dm-murphy** [comment]
@thatblindgeye Thanks for the follow up, Eric! I absolutely want to fix this one and can have a PR ready this week.

**Comment by thatblindgeye** [comment]
Closing this as it has been fixed in #23138
